lib/compiler.js

```diff
@@ -21,13 +21,17 @@ exports.compile = function (code, options) {
 
   const result = { ast: null, code };
 
-  if (! getOption(options, "force") &&  ! importExportRegExp.test(code)) {
+  if (! getOption(options, "force") &&
+      ! importExportRegExp.test(code)) {
+    // Let the caller know the result is no different from the input.
+    options.identical = true;
```

Comment:
There should be a better way to signal that there is no difference since modifying user provided options is generally seen as bad news.